controller/namedspaces_test.go

```diff
@@ -75,7 +76,7 @@ func (rest *TestNamedSpaceREST) TestSuccessQuerySpace() {
 		assert.Fail(t, "Failed to create an identity")
 	}
 
-	name := "Test 24"
+	name := "Test 24" + uuid.NewV4().String()
```

Comment:
I think @hectorj2f had a good idea here; if we start using names that are related to the test its much easier to track where data came from